### PR TITLE
[Docs] Add the Jenkins pipeline section to the project CI/CD integration

### PR DESCRIPTION
**Pull request by xsqian**
@jillnogold Add the Jenkins Pipeline section to the project CI/CD integration part of the docs

### CODE REVIEW

File: docs/projects/ci-integration.md [hunk 1]
```diff
@@ -81,3 +82,78 @@ run:
...
+         steps{
+            auditTools()
+         }
+      }
+      stage('Build') {
+            environment {
+               MLRUN_DBPATH='https://mlrun-api.default-tenant.app.us-sales-341.iguazio-cd1.com'
+               V3IO_ACCESS_KEY=credentials('V3IO_ACCESS_KEY')
+               V3IO_USERNAME='xingsheng'
+               V3IO_API="https://default-tenant.app.us-sales-341.iguazio-cd1.com:8444"
```
Comment (yaronha) [inline docs/projects/ci-integration.md:111]: in new ver (>= 1.0) V3IO_API is determined from the `MLRUN_DBPATH` , can you try running it without setting the `V3IO_API`

File: docs/projects/ci-integration.md [hunk 2]
```diff
@@ -81,3 +82,78 @@ run:
...
+
+void auditTools() {
+   sh '''
+      git version
+      docker version
+   '''
+}
+```
+After the Jenkins pipeline is complete, you will be able to see the MLRun job in the Iguazio Platform GUI like the screenshot below:
+
```
Comment (yaronha) [inline docs/projects/ci-integration.md:156]: replace `Iguazio Platform GUI` with `MLRun UI` (should apply to the open source just as well), also no need to add a new image, OMO we already have that image in under _static

File: docs/projects/ci-integration.md [hunk 3]
```diff
@@ -81,3 +82,78 @@ run:
...
+}
+
+void auditTools() {
+   sh '''
+      git version
+      docker version
+   '''
+}
+```
+After the Jenkins pipeline is complete, you will be able to see the MLRun job in the Iguazio Platform GUI like the screenshot below:
```
Comment (jillnogold) [inline docs/projects/ci-integration.md:155]: After the Jenkins pipeline is complete, you can see the MLRun job in the Iguazio Platform GUI similar to the screenshot below:

File: docs/projects/ci-integration.md (hunk 1 above)
Comment (xsqian) [reply docs/projects/ci-integration.md]: yes, it can run w/o setting the V3IO_API

File: docs/projects/ci-integration.md (hunk 2 above)
Comment (xsqian) [reply docs/projects/ci-integration.md]: edited and removed the image

File: docs/projects/ci-integration.md (hunk 3 above)
Comment (xsqian) [reply docs/projects/ci-integration.md]: changed the wording

Comment (jillnogold) [reply docs/projects/ci-integration.md]: Can you just add period at the end of the two last sentences:

- ...MLRun UI**.**
- See the full example in <link>**.**

Thanks

**Pull request merged**